docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -110,11 +110,10 @@ public class Gift {
 <2> Mark your CDI bean method as `@Transactional` and the `EntityManager` will enlist and flush at commit.
 
 To load some SQL statements when Hibernate ORM starts, add a `import.sql` in the root of your resources directory.
-It contains SQL DML statements (one by line).
-This is useful to have a data set ready for your tests or demos.
+It contains SQL DML statements terminated by a semicolon. This is useful to have a data set ready for your tests or demos.
```

Comment:
Ah ah, no. This is done on purpose: at most one sentence per line. It's better for diffs when updating the doc.

Keep in mind that Asciidoc will append them all on one line.

Reply:
Happens each time I touch an AsciiDoc. It is my IDE I guess :-) Will revert and keep the desirable change. Also add some documentations as suggested by the review.